src/test/scala/sigmastate/SoftForkabilitySpecification.scala

```diff
@@ -0,0 +1,170 @@
+package sigmastate
+
+import org.ergoplatform.ValidationRules.CheckValidOpCode
+import org.ergoplatform._
+import sigmastate.Values.{NotReadyValueInt, ErgoTree, IntConstant}
+import sigmastate.helpers.{ErgoLikeTestProvingInterpreter, ErgoLikeTestInterpreter}
+import sigmastate.interpreter.Interpreter
+import sigmastate.interpreter.Interpreter.ScriptNameProp
+import sigmastate.serialization._
+import sigmastate.lang.Terms._
+import sigmastate.lang.exceptions.{SerializerException, InvalidOpCode, CosterException}
+import sigmastate.serialization.OpCodes.{LastConstantCode, OpCode}
+import special.sigma.SigmaTestingData
+
+class SoftForkabilitySpecification extends SigmaTestingData {
+
+  implicit lazy val IR = new TestingIRContext
+  lazy val prover = new ErgoLikeTestProvingInterpreter()
+  lazy val verifier = new ErgoLikeTestInterpreter
+  lazy val invalidPropV1 = compile(Interpreter.emptyEnv,
+    """{
+     |  HEIGHT > 100 && OUTPUTS.size == 1
+     |}""".stripMargin).asBoolValue
+  lazy val invalidTxV1 = createTransaction(createBox(100, invalidPropV1.asSigmaProp, 1))
+  lazy val invalidTxV1bytes = invalidTxV1.messageToSign
```

Comment:
`.bytes` should be here (and later), instead of `.messageToSign`

Reply:
ErgoLikeTransation doesn't have `bytes`, its id is computed like this
`  override lazy val id: ModifierId = Blake2b256.hash(messageToSign).toModifierId`